### DISCUSSION TITLE
Support NO_COLOR

### DISCUSSION THREAD

**Discussion by rolandtritsch** [com-lihaoyi/mill#4170] [Ideas]
If you are using mill (with munit) in VSCode the build (and testing) output will (try to and fail to) display ANSI color codes in the `Debug Console`. munit supports the [NO_COLOR](https://no-color.org) env var. Mill obviously has the `--color=false` flag, but that means that I have to configure two things (instead of one).

If mill would support the NO_COLOR env var then it would all "just work".

The order of processing should be NO_COLOR first and the --color flag second (means with --color you can override NO_COLOR).

Happy to contribute a PR.

Thoughts?

**Comment by gamlerhart** [comment]
PR: https://github.com/com-lihaoyi/mill/pull/4246
